crates/uv-python/src/interpreter.rs

```diff
@@ -731,13 +754,20 @@ pub struct BrokenSymlink {
     pub path: PathBuf,
     /// Whether the interpreter path looks like a virtual environment.
     pub venv: bool,
+    /// Whether the link is a Windows trampoline target
+    pub trampoline_target: bool,
 }
 
 impl Display for BrokenSymlink {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let symlink_term = if self.trampoline_target {
+            "Missing trampoline target"
```

Comment:
I'm not sure if we want to expose the term "trampoline" to users. But "Broken symlink" seemed misleading since in the trampoline case, it will actually be a junction-containing path.